### DISCUSSION TITLE
Reqnroll Rider plugin v2024.1.1 published

### DISCUSSION THREAD

**Discussion by gasparnagy** [reqnroll/Reqnroll#86] [Announcements]
Good news! The first version of our Rider plugin has been published to the JetBrains marketplace!

https://plugins.jetbrains.com/plugin/24012-reqnroll-for-rider

Thanks for everyone who contributed in creating this, especially @Socolin who did a great job to make this possible. 💯 

Please note, that cucumber expressions are not yet supported with the plugin, but hopefully we will have that as well soon.

**Comment by DerAlbertCom** [comment]
Should that PlugIn work?

For me, it works neither with the Reqnroll quickstart nor with a small to Regnroll converted Test-project (without SpecFlow Compat NuGet). 

- It hides all feature files in the solution view
- In the Filesystem View, I can see the files, but their step bindings are not found.

I'm willing to help, I downloaded the source, compiled it. Started in the IDE, but I have no clue where to start.

**Reply by ajeckmans** [reply]
You should go over to its repository :)

You're running into this issue: https://github.com/reqnroll/Reqnroll.Rider/issues/1
The workaround that is described there should make everything work.

**Reply by DerAlbertCom** [reply]
Yeah... Workaround. I'm more for fixing the issue, will try the workaround until then. Still willing to help. But it is not easy to get an understanding of the Plugin.

**Reply by ajeckmans** [reply]
I know :) I'm just saying that it is not quite clear what caused this issue. It surfaces only in Rider, so I'm all for saying it is related to that plugin. Though it is not quite sure where the issue lies. In either case I'd suggest debugging that plugin first (though it can be tedious work). @Socolin did previously state he welcomes help so maybe he can chip in with some pointers on how to develop for the plugin.

**Reply by Socolin** [reply]
I did not have time yet to check why this is not working. It should be automatic, probably a details. I'll check soon, if someone wants to take a look ./gradlew :runIde then from rider attach to rider backend and you can put breakpoints to find why the files are not iterated over.

I'll try soon, at least when rider 2024.1 comes out I'll probably upgrade all my projects to reqnroll and fix the issue
